src/applications/static-pages/download-1095b/mocks/script/utils.js

```diff
@@ -0,0 +1,15 @@
+/**
```

Comment:
This whole file borrowed from Profile and condensed for specific usage in this app.